### ISSUE TITLE
Is classes and test-classes on the classpath for locating/reading classpath: urls?

### ISSUE THREAD

**Issue by warrenc5** [mojohaus/properties-maven-plugin#52]
I want to load a processed resource from target/test-classes/

The <url>classpath:application.properties</url> doesn't seem to work.

Run maven with -X doesn't show the classpath in debug.

What makes me think resources will be on the project classpath anyway? 

Would I use build helper to add target/classes to the classpath?

**Comment by slawekjaranowski** [comment]
In current implementation only plugin classpath is recognized ... 

We need build a new classloader based on project targets and project dependencies.

**Comment by warrenc5** [comment]
Yes, I see some precursor work has been done here in this PR, perhaps this can be reused/extended?

https://github.com/mojohaus/properties-maven-plugin/pull/42
